blocks/small-screen-cta/small-screen-cta.css

```diff
@@ -0,0 +1,47 @@
+.call-to-action {
+  margin-block-end: 4rem;
```

Comment:
Should we be adding the bottom margin on these blocks individually? The margin underneath looks like it's the global margin between all of the sections. I'm not sure if that's something we've handled yet:
![Screenshot 2025-05-21 at 10 58 39 AM](https://github.com/user-attachments/assets/cc366776-84ef-4c11-9314-56cccd197200)

Reply:
Yeah I'm not sure! We've been a little inconsistent. This block won't exist inside a layout, but it is something to discuss once the pages start coming together.